### PR TITLE
Tinymce

### PR DESCRIPTION
**Pull request by RachellCalhoun**
I set up tinymce for text fields and flatpages. Seems to be working. check and merge if it looks good.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-coverage==4.0.3
```
Comment (RachellCalhoun) [inline requirements.txt:1]: Coverage? What is this and sorry I deleted it. should I add it in?

**Comment by RachellCalhoun** [comment]
@jinpark this is the same as i talked with you on slack. I had  a question about requirements though. can you check please?

**Comment by hassanabidpk** [comment]
I have to try it first :person_frowning:

**Comment by jinpark** [comment]
I'll let @hassanabidpk review since I was the one that helped out :)

File: requirements.txt (hunk 1 above)
Comment (jinpark) [inline requirements.txt:1]: This is somethign that hassan added in a previous PR. You should add it back

**Comment by jinpark** [comment]
I'm going to merge this in for now to move things along.

@hassanabidpk please review even though this is merged.

**Pull request merged**